client/order/index.js

```diff
@@ -129,102 +134,176 @@ jQuery( function ( $ ) {
-		dispute.amount,
-		dispute.currency
-	);
+	let refundDisabledNotice = '';
+	if ( shouldDisableRefund ) {
+		const refundButton = document.querySelector( 'button.refund-items' );
+		if ( refundButton ) {
+			disableRefund = true;
 
-	let urgency = 'warning';
-	let buttonLabel = __( 'Respond now', 'woocommerce-payments' );
-	let suffix = '';
-
-	let titleText = isInquiry( dispute )
-		? titleStrings.inquiry_default
-		: titleStrings.dispute_default;
-
-	// If the dispute is due within 7 days, use different wording.
-	if ( countdownDays < 7 ) {
-		titleText = isInquiry( dispute )
-			? titleStrings.inquiry_urgent
-			: titleStrings.dispute_urgent;
-
-		suffix = sprintf(
-			// Translators: %s is the number of days left to respond to the dispute.
-			_n(
-				'(%s day left)',
-				'(%s days left)',
-				countdownDays,
+			// Disable the refund button.
+			refundButton.disabled = true;
+
+			const disputeDetailsLink = getDetailsURL( dispute.id, 'disputes' );
+			// TODO: finalize text copy after we come up with the final decision.
+			refundDisabledNotice = createInterpolateElement(
+				__(
+					'Refunds and order editing are disabled because this order was disputed. <a>View dispute</a>',
+					'woocommerce-payments'
+				),
+				{
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					a: <a href={ disputeDetailsLink } />,
+				}
+			);
+			let tooltipText = __(
+				'Refunds and order editing are disabled because this order was disputed.',
 				'woocommerce-payments'
-			),
-			countdownDays
-		);
-	}
+			);
 
-	const title = sprintf(
-		titleText,
-		amountFormatted,
-		reasons[ dispute.reason ].display,
-		dateI18n( 'M j, Y', dueBy.local().toISOString() )
-	);
+			if ( isAwaitingResponse( dispute.status ) ) {
+				refundDisabledNotice = __(
+					'Refunds and order editing are disabled during disputes.',
+					'woocommerce-payments'
+				);
+				tooltipText = refundDisabledNotice;
+			} else if ( isUnderReview( dispute.status ) ) {
+				refundDisabledNotice = createInterpolateElement(
+					__(
+						'This order has a payment dispute. Refunds and order editing are disabled during disputes. <a>View dispute</a>',
+						'woocommerce-payments'
+					),
+					{
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						a: <a href={ disputeDetailsLink } />,
+					}
+				);
+				tooltipText = __(
+					'Refunds and order editing are disabled during disputes.',
+					'woocommerce-payments'
+				);
+			}
 
-	// If the dispute is due within 72 hours, we want to highlight it as urgent/red.
-	if ( countdownDays < 3 ) {
-		urgency = 'error';
+			// Change refund tooltip's text copy.
+			for ( const maybeSibling of refundButton.parentNode.children ) {
+				if ( maybeSibling === refundButton ) {
+					continue;
+				}
+				const tooltip = maybeSibling.querySelector(
+					'.woocommerce-help-tip'
+				);
+				if ( ! tooltip ) {
+					continue;
+				}
+				jQuery( tooltip ).tipTip( { content: tooltipText } );
+			}
```

Comment:
Minor suggestion: this might be more readable if we handle all of the logic with jQuery, since we're using jQuery for `tipTip` anyway – happy to stick with your vanilla method though.

I also notice that the `aria-label` attribute should also be updated to include the new tooltip content.

```suggestion
			jQuery( refundButton )
				.parent()
				.find( '.woocommerce-help-tip' )
				.attr( {
					// jQuery.tipTip uses the title attribute to generate the tooltip.
					title: tooltipText,
					'aria-label': tooltipText,
				} )
				// Regenerate the tipTip tooltip.
				.tipTip();
```

Reply:
Good idea. Addressed in https://github.com/Automattic/woocommerce-payments/pull/7043/commits/54b7199da469a483e4c7b22ed064704b379b3158.